linodecli/plugins/obj/helpers.py

```diff
@@ -120,13 +120,8 @@ def _denominate(total):
 # helper functions for output
 def _borderless_table(data):
     """
-    Returns a terminaltables.SingleTable object with no borders and correct padding
+    Returns a rich.Table object with no borders and correct padding
     """
-    tab = SingleTable(data)
-    tab.inner_heading_row_border = False
-    tab.inner_column_border = False
-    tab.outer_border = False
-    tab.padding_left = 0
-    tab.padding_right = 2
+    tab = Table(show_header=False, show_edge=False, padding=(0, 2, 0, 2))
```

Comment:
This differs a bit from the current format, and you need to add the rows in:

```suggestion
    tab = Table(show_header=False, show_edge=False, padding=(0, 2, 0, 2), box=None)
    for row in data:
        row = [Text.from_ansi(str(item)) for item in row]
        tab.add_row(*row)
```

I converted `item` to a string here, but that creates very verbose date strings - some special formatting for them is probably warranted too.

Reply:
Can you explain your steps to test this module so I can take a look for myself?

Reply:
See my suggestion above to add `rich.Text` import to accommodate this.

Reply:
Yeah, my full diff on this file was:

```diff
diff --git a/linodecli/plugins/obj/helpers.py b/linodecli/plugins/obj/helpers.py
index b3a5883..ce247e0 100644
--- a/linodecli/plugins/obj/helpers.py
+++ b/linodecli/plugins/obj/helpers.py
@@ -5,6 +5,7 @@ from argparse import ArgumentTypeError
 from datetime import datetime

 from rich.table import Table
+from rich.text import Text

 from linodecli.plugins.obj.config import DATE_FORMAT

@@ -122,6 +123,9 @@ def _borderless_table(data):
     """
     Returns a rich.Table object with no borders and correct padding
     """
-    tab = Table(show_header=False, show_edge=False, padding=(0, 2, 0, 2))
+    tab = Table(show_header=False, show_edge=False, padding=(0, 2, 0, 2), box=None)
+    for row in data:
+        row = [Text.from_ansi(str(item)) for item in row]
+        tab.add_row(*row)

     return tab
```

I wasn't testing with a script, just installed the cli from source (`pip install -e .` in the root of this repo) and ran some commands; `linode-cli obj ls` was enough to see that no data was displayed, and `linode-cli ls $BUCKET_NAME` to show the contents of a bucket (using the same output formatting).  If you don't have object storage set up, `linode-cli obj mb $BUCKET_NAME` will do that for you and create a new bucket (but I think there's a nominal charge for enabling the service).